boards/arc/nsim/support/mdb_hs_smp.args

```diff
@@ -49,4 +49,3 @@
 	-on=nsim_print-sys-arch
 	-noprofile
 	-nogoifmain
-	-instrs_per_pass=10
```

Comment:
As we discussed earlier it's better to explicitly set this option as we have different default values for GUI & command line mode.